### DISCUSSION TITLE
Layers names are not up-to-date with Yocto

### DISCUSSION THREAD

**Discussion by hurtadosanti** [meta-rust/meta-rust#432] [Q&A] [answered]
According to the documentation there should be a layer for each release, but I dont see the latest LTE codenames, is there an upgrade or a decision here? thanks

**Comment by srwalter** [comment, marked as answer]
We should probably update the README.  What happens in practice is we use the master branch for all Yocto releases, until there is some incompatible change.  As you can see in our layer.conf, the master branch still supports all versions of Yocto since dunfell.  Additionally our CI verifies dunfell, kirkstone, and mickledore.

**Reply by kraj** [reply]
some layers use that process so its fine once documented.

**Comment by daleCBT** [comment]
Just found this.  New to yocto and trying to add Rust to (almost) latest Dunfell.  

I see that there is now a `dunfell` branch for `meta-rust`.  Does what I'm reading above mean that I should _not_ use `master`, now that there's a `dunfell` branch for `meta-rust`?

**Reply by srwalter** [reply]
Yes, that's correct.  There are changes now in master that won't work with dunfell.

**Reply by daleCBT** [reply]
Amazing. Thank you.
